aptos-move/framework/aptos-framework/sources/block.move

```diff
@@ -108,7 +107,7 @@ module aptos_framework::block {
         // transition is the last block in the previous epoch.
         stake::update_performance_statistics(proposer_index_optional, failed_proposer_indices);
 
-        if (timestamp - reconfiguration::last_reconfiguration_time() > block_metadata_ref.epoch_interval) {
+        if (timestamp - reconfiguration::last_reconfiguration_time() >= block_metadata_ref.epoch_interval) {
```

Comment:
This is not a big problem but a block exactly an epoch interval from the last epoch start should trigger a new epoch. Otherwise, each epoch would always last ~1 block longer than epoch interval.